cli/onionshare_cli/web/web.py

```diff
@@ -385,3 +385,15 @@ def cleanup(self):
             dir.cleanup()
 
         self.cleanup_tempdirs = []
+
+    def waitress_custom_shutdown(self):
+        """Shutdown the Waitress server immediately"""
+        # Code borrowed from https://github.com/Pylons/webtest/blob/4b8a3ebf984185ff4fefb31b4d0cf82682e1fcf7/webtest/http.py#L93-L104
+        self.waitress.was_shutdown = True
```

Comment:
This `was_shutdown` property is usually used to check for exception in waitress.run part of code (https://github.com/evandroforks/anki/commit/14d4832ce8f4c026fe81c6466d81606c7e5f2d18#diff-9904faac0b4144bb89eaa2c1205913d0b1322b0476d4152c1720255f57d781c5R64-R66) (https://github.com/Pylons/webtest/blob/main/webtest/http.py#L88-L90) . Should we also put `create_server` and `waitress.run()` inside a try-except block? I am not really sure when the exception would happen which the codes are trying to handle. Honestly, if we don't need the try-except block, we might as well remove this line.